app/services/audio_processing_service.py

```
@@ -177,98 +194,149 @@ def process_audio_task(
+                        "error": None,
+                        # Written here as well as on the requeue, because the
+                        # requeue write is guarded and can be skipped during
+                        # the very outage being retried through. Without this a
+                        # task that recovered would report zero retries.
+                        "retry_count": attempt - 1,
+                    },
+                )
+
+                return
+
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)